esx_service/cli/vmdkops_post_update.11.1.py

```diff
@@ -0,0 +1,192 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -------------------------------------------------------------------------------------------
+# Updates ESX with vSphere Docker Volume Service 0.11 (and earlier)
+#  to 0.11.1 and further
+#
+# This code expects to be places in  /usr/lib/vmware/vmdkops/bin/ to pick up correct modules.
+# -------------------------------------------------------------------------------------------
+
+#import os
+import os.path
+import sqlite3
+import sys
+import shutil
+
+import vmdk_ops
+# vmdkops python utils are in PY_LOC, so add to path.
+sys.path.insert(0, vmdk_ops.PY_LOC)
+
+import vmdk_utils
+import auth
+import auth_data_const
+import auth_data
+from error_code import ErrorCode, error_code_to_message
+
+# Hard coded (in auth package) UUD for default tenant.
+STATIC_UUID = auth.DEFAULT_TENANT_UUID
+STATIC_NAME = auth.DEFAULT_TENANT
+
+# CLI return codes
+OK = 0
+ERROR = 1
+
+
+def main():
+    """
+    This code updates ESX with vSphere Docker Volume Service 0.11 (and earlier)
+    to 0.11.1 and further, by moving _DEFAULT tenant ID to well known and static UUID,
+    and then correcting directories layout and auth_db tables to comply with new UUID.
+
+    Specifically, it does the following:
+    - Checks if AUTH_DB exists.
+      If it does not, exit with a message - it means nothing to patch on this ESX
+    - Gets uuid (aka "old_uuid') for _DEFAULT tenant from DB.
+      If it already STATIC_UUID , exit with a message - nothing to patch
+    - Stops the service
+    - backs up the DB
+    - scans through all <datastore>/volumes/dockvols and
+        - mkdir STATIC_UUID, if it does not exist
+        - move all from old_uuid to STATIC_UUID
+        - symlinks "_DEFAULT" to STATIC_UUID
+    In single DB transcation
+        - replaces old_uuid with STATIC UUID in tenant_id field for all tables:
+          (privileges, vms,  tenants, volumes)
+    starts the service , and if all good removes backup DB
+
+    NOTE: this does not delete any data, so the Docker volumes will stay around
+          no matter if the code succeeds or fails
+    """
+
+    stop_service = True  # 'False' here makes debugging faster.
+
+    dbfile = auth_data.AUTH_DB_PATH
+
+    # STEP: check DB presense and fetch old_uuid
+    if not os.path.isfile(dbfile):
+        print("Config DB", dbfile, "is not found, nothing to update - exiting.")
+        sys.exit(0)
+
+    cursor = sqlite3.connect(dbfile).cursor()
+    cursor.execute("select * from tenants where name='{0}'".format(STATIC_NAME))
+    try:
+        tenant_id, tenant_name, tenant_desr, tenant_def_db = cursor.fetchone()
+    except TypeError:
+        print("Can't find '{0}' tenant, exiting".format(STATIC_NAME))
+        sys.exit(ERROR)
+
+    print("Found default tenant: {0} {1} {2} {3}".format(tenant_id,
+                                                         tenant_name, tenant_desr, tenant_def_db))
```

Comment:
nit: can you please rename "tenant_def_db" to "tenant_def_ds" for better reading?